pkg/skaffold/build/local/util.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local
+
+import (
+	"os"
+)
+
+func isFile(filePath string) bool {
+	info, error := os.Stat(filePath)
```

Comment:
`s/error/err/`. `error` is a type. The compiler understands you use it as a variable but the reader will need a second to understand what's going on.